src/actions/getRoomInfo.js

```diff
@@ -0,0 +1,26 @@
+import { getRoomsPromise } from './helpers/getRoomsHelper';
+import { getPlayersPromise } from './helpers/getPlayersHelper'
+
+const getRoomInfo = () => {
+  return async function (dispatch) {
+    try {
+      const roomIds = await getRoomsPromise();
```

Comment:
It's better to modify function name from `getRoomsPromise` to `getRoomIdsPromise`.